src/helpers/map.js

```diff
@@ -209,13 +209,10 @@ export function init2dArray(width, height, defaultVal) {
  * m2: the second matrix (of the same dimensions as m1)
  */
 export function multiplyCorrespondingElementsAndSum(m1, m2) {
-  let condition = (m1.length === m2.length);
-  let errorMessage = 'multiplyCorrespondingElementsAndSum: m1 and m2 are not the same width';
-  utils.assert(condition, errorMessage);
-
-  condition = (m1[0].length === m2[0].length);
-  errorMessage = 'multiplyCorrespondingElementsAndSum: m1 and m2 are not the same height';
-  utils.assert(condition, errorMessage);
+  assert(m1.length === m2.length,
+    'multiplyCorrespondingElementsAndSum: m1 and m2 are not the same width');
+  assert(m1[0].length === m2[0].length,
+    'multiplyCorrespondingElementsAndSum: m1 and m2 are not the same height');
```

Comment:
Does the linter not flag these?  For multiline things put each parameter on its own line [(style guide link)](https://github.com/airbnb/javascript#functions--signature-invocation-indentation)